mapclientplugins/geometricfitstep/view/geometricfitwidget.py

```diff
@@ -476,8 +476,8 @@ def _setupConfigWidgets(self):
         self._ui.configSettingGroup_fieldChooser.setNullObjectName("-")
         self._ui.configSettingGroup_fieldChooser.setConditional(fieldIsManagedGroup)
         self._ui.configSettingGroup_fieldChooser.setField(Field())
-        self._ui.configDataWeight_checkBox.setVisible(False)
-        self._ui.configDataWeight_lineEdit.setVisible(False)
+        self._ui.configDataWeight_checkBox.setDisabled(True)
+        self._ui.configDataWeight_lineEdit.setDisabled(True)
```

Comment:
Minor, but use alphabetical order, so DataWeight is after DataProportion, esp. since that is also the order in the UI widgets.